app/components/bali/calendar/index.scss

```
@@ -0,0 +1,89 @@
+$td-padding: 6px;
+$td-border-color: #ddd;
+$w-100: 100%;
+$grey-7: hsl(200, 13%, 95%) !default;
+$grey-alpha: hsla(0, 0%, 35%, 0.5);
+$grey-darker-alpha: hsla(224, 28%, 19%, 0.4);
+
+.calendar-component {
+  td {
+    padding: $td-padding;
+    vertical-align: top;
+    border: 1px solid $td-border-color;
+  }
+
+  .day {
+    position: relative;
+    width: $w-100;
```

```suggestion
    width: $w-100;
```
Im think this can be overrided by using `is-fullwidth` from bulma.

---

it seems that the `is-fullwidth` class does not have an effect on `td` tag